src/element.ts

```diff
@@ -43,16 +42,30 @@ export abstract class Element {
     return `auto${Element.ID++}`;
   }
 
+  /**
+   * Default font for text. This is not related to music engraving. Instead, see `Flow.setMusicFont(...fontNames)`
+   * to customize the font for musical symbols placed on the score.
+   */
+  static TEXT_FONT: Required<FontInfo> = {
+    family: Font.SANS_SERIF,
+    size: Font.SIZE,
+    weight: FontWeight.NORMAL,
+    style: FontStyle.NORMAL,
+  };
+
   private context?: RenderContext;
   protected rendered: boolean;
   protected style?: ElementStyle;
   private attrs: ElementAttributes;
   protected boundingBox?: BoundingBox;
   protected registry?: Registry;
 
-  // fontStack and musicFont are both initialized by the constructor via this.setFontStack(...).
-  protected fontStack!: Font[];
-  protected musicFont!: Font;
```

Comment:
Removed the per-element music font. It's a nice idea but it worked in unexpected ways in all of my tests. Having one global font stack is simpler for now. Maybe we can revisit this someday if people desire this feature.